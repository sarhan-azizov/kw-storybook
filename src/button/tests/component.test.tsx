import React from 'react';
import { shallow } from 'enzyme';

import { Button } from '../index';

describe('Button component', () => {
    it('should render component with default props', () => {
        const wrapper = shallow(<Button title="123" />);

        expect(wrapper).toMatchSnapshot();
    });
});
