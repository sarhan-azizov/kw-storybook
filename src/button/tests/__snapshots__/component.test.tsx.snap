// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render component with default props 1`] = `
<button
  type="button"
>
  123
  <span />
</button>
`;
